pkg/generator/ingress_translator.go

```diff
@@ -313,7 +310,7 @@ func createUpstreamTLSContext(caCertificate []byte, activatorSAN string, alpnPro
 					},
 					MatchSubjectAltNames: []*envoymatcherv3.StringMatcher{{
 						MatchPattern: &envoymatcherv3.StringMatcher_Exact{
-							Exact: activatorSAN,
+							Exact: certificates.FakeDnsName,
```

Comment:
I guess if we ever have multiple-knative installations this will have to be changed?